Code review (#29)
cBridges851/poseidon-text-adventure: README.md

# Poseidon-Text-Adventure 🐙

> A text adventure created by memebers of Poseiden for coding challenge 3.

Reviewer: ```suggestion
A text adventure created by members of Poseidon for coding challenge 3.
```

> # Context 📑

Reviewer: Subheading of poseidon-text-adventure
```suggestion
## Context 📑
```

You start the game inside PebbleTown and you can travel around the area finding suprises along the way. Should you travel in the right direction you may come across a field, enter if
you dare..... Should you go south! I wish you the best of luck you brave soul.

> # How to play 🔌

Reviewer: Subheading of poseidon-text-adventure
```suggestion
## How to play 🔌
```

To start the game open the game folder and run the python file "adventure_game.py". 
- If you are new to the game, select a the new option and enter your name before you are transported to the main square to play the main game. 
- If you aren't new, select the returning player option and use the name you've previously used.

> # Help 🆘

Reviewer: Subheading of poseidon-text-adventure
```suggestion
## Help 🆘
```

Should you need any help whilst in the game, type help when in any of the main areas to bring up the help menu.

> # Credits 🎞

Reviewer: Subheading of poseidon-text-adventure
```suggestion
## Credits 🎞
```

> - John Mason

Reviewer: Might be nice to have profile links
```suggestion
- [John Mason](https://github.com/johnmason27)
```

> - Christa Bridges

Reviewer: Might be nice to have profile links
```suggestion
- [Christa Bridges](https://github.com/cBridges851)
```

> - Joshua Talbot

Reviewer: ```suggestion
- [Joshua Talbot](https://github.com/13JoshT)
```
